components/host-dashboard/ScheduledExpensesBanner.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useQuery } from '@apollo/client';
+import { create, Mode } from '@transferwise/approve-api-action-helpers';
+import { FormattedMessage } from 'react-intl';
+
+import { addAuthTokenToHeader } from '../../lib/api';
+import { API_V2_CONTEXT, gqlV2 } from '../../lib/graphql/helpers';
+
+import ConfirmationModal from '../ConfirmationModal';
+import { expensesListAdminFieldsFragment, expensesListFieldsFragment } from '../expenses/graphql/fragments';
+import { Box, Flex } from '../Grid';
+import TransferwiseIcon from '../icons/TransferwiseIcon';
+import MessageBox from '../MessageBox';
+import StyledButton from '../StyledButton';
+import { TOAST_TYPE, useToasts } from '../ToastProvider';
+
+const scheduledExpensesQuery = gqlV2/* GraphQL */ `
+  query HostDashboardScheduledExpenses(
+    $hostId: String!
+    $limit: Int!
+    $status: ExpenseStatusFilter
+    $payoutMethodType: PayoutMethodType
+  ) {
+    expenses(host: { id: $hostId }, limit: $limit, status: $status, payoutMethodType: $payoutMethodType) {
+      totalCount
+      offset
+      limit
+      nodes {
+        ...ExpensesListFieldsFragment
+        ...ExpensesListAdminFieldsFragment
+      }
```

Comment:
It looks like we're only using `totalCount` and `expense.id` in this component, maybe we can reduce the number of fields retrieved in this query?